serverless_llm/serve/backends/transformers_backend.py

```diff
@@ -70,10 +71,9 @@ async def init_backend(self) -> None:
                 )
                 torch_dtype = torch.float16
             storage_path = os.getenv("STORAGE_PATH", "./models")
-            # TODO: storage_path = os.path.join(storage_path, "transformers")
-
+            model_path = Path(os.path.join(storage_path, "transformers", self.model_name)).resolve()
             self.model = load_model(
-                self.model_name,
+                str(model_path),
```

Comment:
No, it will not work. Since the storage path inside serverless llm store server is still `./models`, so it will only search for `./models/{model_name}` but we save it at `./models/transformers/{model_name}`. 
The log of the server will be
```
I20240722 12:20:36.171752 57148 checkpoint_store.cpp:76] Model facebook/opt-1.3b is registered
I20240722 12:20:36.171855 57148 model.cpp:37] Tensor file "./models/facebook/opt-1.3b/tensor.data_0" does not exist
E20240722 12:20:36.171905 57148 model.cpp:49] Model facebook/opt-1.3b does not exist
```

Reply:
The `storage_path` parameter in `load_model` is only used for the client to check the metadata path, and it won't work for server side.

Reply:
OK, makes sense to me. But why would an absolute path work? like `model_path = Path(os.path.join(storage_path, "transformers", self.model_name)).resolve()`

Reply:
Since both client and server are running on the same machine, and both of them accept path as input to load model, so absolute path will be work here.